modules/misc/meta.nix

```diff
@@ -0,0 +1,54 @@
+{ defaultPkgs, lib, ... }:
+with lib;
+{
+  imports = [
+    # TODO: to have modulesPath we'd have to define it using specialArgs
+    # (Can't use config._module.args to define imports)
+    (defaultPkgs.path + "/nixos/modules/misc/meta.nix")
```

Comment:
Doing this means `defaultPkgs` must be evaluated even if `_module.args.pkgs` is overridden, but I don't see an alternative.

It's no different to what we do currently though, so finding a solution doesn't block this PR.